### ISSUE TITLE
Second example, second nested bullet not correct for 3.3.3: Error Suggestion

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1799]
https://www.w3.org/WAI/WCAG22/Understanding/error-suggestion

The second example, second nested bullet, goes like:

> - **Suggestions from a Limited Set of Values**
> An input field requires that a month name be entered. If the user enters "12," suggestions for correction may include
>       - A description of the set of values, e.g., "Please provide the name of the month."

Now it first says: "An input field requires that a month name" (done by a label? "Name of month"? only Month Name?)
Then the 'fix'... "A description of the set of values"... 
BUT then it provide the 'good' example of "Please provide the name of the month." ?!

Isn't that not like exactly almost the label and gives no extra suggestions?

This example should be deleted as it gives a false idea of a fix!

**Comment by patrickhlauke** [comment]
https://github.com/w3c/wcag/pull/1804
